templates/webapi/test/overview.html.ep

```diff
@@ -42,8 +42,12 @@
         <div class="card-body">
             % my $allow_commenting = @$job_ids && current_user;
             % if ($allow_commenting) {
-            <button type="button" class="btn btn-secondary btn-circle btn-sm trigger-edit-button" onclick="showAddCommentsDialog()" title="Add comments">
-                <i class="fa fa-comment" aria-hidden="true"></i>
+            <button type="button" class="btn btn-secondary btn-circle btn-sm
+            trigger-edit-button" onclick="showAddCommentsDialog()" title="Restart or comment jobs">
+                <span class="fa-stack" style="font-size: 0.9em" >
+		  <i class="fa fa-comment fa-stack-2x text-danger-info" aria-hidden="true"></i>
+		  <i class="fa fa-undo fa-stack-1x" style="color: #0c4374;" aria-hidden="true"></i>
```

Comment:
Maybe we should add a CSS class for this color instead of defining it in HTML. Maybe we can then also use some existing CSS color values defined by Bootstrap or in our SCSS theme. You should also check whether this looks good in darkmode.

Reply:
Checked in the dark mode and looks good. 
styles moved to ` overview.scss`